frontend/src/pages/LandingPage/Landingpage.tsx

```diff
@@ -38,14 +43,56 @@ const Landing = (props: RouteComponentProps) => {
         />
         <h2>Simplify managing your rental properties.</h2>
       </CTA>
-        <Footer>
-          <a href='/'><Logo src={lodgel} alt='Lodgel logo' /></a>
-          <a href='https://example.com'>Email</a>
-          <a href='https://twitter.com'>Twitter</a>
-          <a href='https://github.com/Lambda-School-Labs/labs9-cleaner-pos'>
-            GitHub
-          </a>
-        </Footer>
+      <HeroContainer>
+        <DescContainer>
+          <SectionHeader>Properties</SectionHeader>
+          <Description>
+            <DescHeader>Manage your Properties</DescHeader>
+            <p>
+              A detailed view for each of your properties is at your fingertips.
+              You have the freedom to add and remove properties as your
+              portfolio changes over time, which keeps complete control of
+              how your properties are managed in your hands.
+            </p>
+          </Description>
+        </DescContainer>
+        <Screenshot src={house} alt='house' />
+      </HeroContainer>
+      <HeroContainer>
+        <SectionHeader>Guests</SectionHeader>
+        <Description>
+          <DescHeader>Keep Tabs on you Guests</DescHeader>
+          <p>
+            Your guests are the heart of your business. To help you deliver
+            high-quality service that your guests deserve, we have created customizable
+            checklists that ensures all of your guests' needs are met.
+          </p>
```

Comment:
I would say "To help you deliver **the** high quality service that your guests deserve." This is pretty nitpicky, though.

Reply:
Also, the checklists ensure, not ensures.

Reply:
Good catches 😁